yt/frontends/owls/tests/test_outputs.py

```diff
@@ -54,3 +55,19 @@ def test_snapshot_033():
 @requires_file(os33)
 def test_OWLSDataset():
     assert isinstance(data_dir_load(os33), OWLSDataset)
+    
+    
+@requires_ds(os33)
+def test_OWLS_particlefilter(os33):
```

Comment:
This function shouldn't take any arguments, this should look like:

```
def test_OWLS_particlefilter():
```

That should fix the test failure.